backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -0,0 +1,57 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+import org.hibernate.annotations.Immutable;
+import org.hibernate.annotations.Type;
+
+import gov.cdc.usds.simplereport.db.model.auxiliary.TestResult;
+
+@Entity
+@Immutable
+public class TestEvent extends AuditedEntity {
```

Comment:
Right now this has no foreign keys, which is probably a mistake. Probably should have an explicit link to `Person` and possibly one to `Organization`, though obviously that one is implicit as well.